config/pycats_config.ini

```diff
@@ -10,12 +10,22 @@ validate_body = True
 validate_is_field_missing = True
 
 [web]
-webdriver_folder = D:\Web Driver\
-selenium_server_executable = D:\Web Driver\selenium-server-standalone-3.141.59.jar
-chrome_driver_name = chromedriver.exe
+webdriver_folder = /Users/sdi/PycharmProjects/snoo-test-automation/framework/web_driver/common/
+selenium_server_executable = /Users/sdi/PycharmProjects/snoo-test-automation/framework/web_driver/common/selenium-server-standalone-3.14.0.jar
```

Comment:
fixed

Reply:
@maximrodin 
probably we need to set some unused data instead of specified version of server. This should show that we need to fill in these fields before running tests?